model/project_ref.go

```diff
@@ -34,30 +35,24 @@ import (
 // The ProjectRef struct contains general information, independent of any
 // revision control system, needed to track a given project
 type ProjectRef struct {
```

Comment:
Nothing in this section changed logically; I just rearranged given the comments used to be specific to what could / could not be defined by repo/branch.